src/stylesheets/components/_site-search.scss

```diff
@@ -54,6 +54,33 @@ $icon-size: 40px;
 .app-site-search__wrapper {
   display: block;
   position: relative;
+
+  // Adds a 'fake border' that's shorter than the width of the menu to create
+  // separation between the input and search items.
+  // This is here on the wrapper element and absolute positioned under the input
+  // rather than on the menu because we can't move a pseudo element out of the
+  // menu eg: using minus margins or absolute positioning without breaking the
+  // menu's overflow conditions.
```

Comment:
As a thought on this, was placing the pseudo-element on the menu and increasing the menu's negative top margin to 'move it out' something that was tried? Seems like it'd be a bit neater, though obviously, I don't know if that'd work here.

Reply:
I gave this a go locally to quickly try to remind myself and unforunately it doesn't work. The issue is that the menu's overflow conditions mean that any negitive margin artificial positioning means the element disappears because it's inside an element with `overflow-x: hidden`. This doesn't seem like it would cause problems since we're talking about content in the 'y' direction, but I think this is what's noted on [the overflow-y MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-y):

>If [overflow-x](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-x) is hidden, scroll, or auto and the overflow-y property is visible (default), the value will be implicitly computed as auto.

I tried messing with the overflow so that it would only manage overflow for the bottom of the element and not the top bt couldn't figure it out. I'm open to ideas.

Reply:
Ack, that's rather annoying. Something that visually looks to be part of the dropdown whilst actually being detached from it feels... brittle. Probably hasn't helped with readability of stuff either. 

I don't have a better idea to offer though, so I'll give it a poke assuming this is the best route available to us.

Reply:
Could [`position: fixed`](https://developer.mozilla.org/en-US/docs/Web/CSS/position?v=example#fixed) be helpful to keep the separator within the `<ul>` and at its top? There are a couple of properties that make an object using `position: fixed` not be fixed relative to the page, but a closer [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block).

Reply:
I gave @romaricpascal's `position: fixed` idea a go and unfortuantely the overflow problem comes back. `fixed` with `contain: layout` as an example doesn't let the pseudo element escale the menu parent, so long as those overflow settings are in place. Same results for the other suggestions provided by MDN. I could resolve this by changing the scope to `app-site-search__wrapper` but that's basically what I'm doing now.

Unless there's a way to adjust the overflow settings without breaking the component I'm not sure there's a less weird solution. The actual answer IMO is to put a container around the `ul` which we can do upstream at a later date.